tests/ibmblockcsi_controller_test.go

```diff
@@ -0,0 +1,191 @@
+/**
+ * Copyright 2019 IBM Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers_test
+
+import (
+	"context"
+	"time"
+
+	csiv1 "github.com/IBM/ibm-block-csi-operator/api/v1"
+	"github.com/IBM/ibm-block-csi-operator/pkg/config"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	rbacv1 "k8s.io/api/rbac/v1"
+	storagev1 "k8s.io/api/storage/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+)
+
+var _ = Describe("Controller", func() {
+
+	const timeout = time.Second * 30
+	const interval = time.Second * 1
+	var ibc *csiv1.IBMBlockCSI
+	var ns = "default"
+	var ibcName = "test-ibc"
+
+	BeforeEach(func() {
+		ibc = &csiv1.IBMBlockCSI{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      ibcName,
+				Namespace: ns,
+			},
+			Spec: csiv1.IBMBlockCSISpec{
+				Controller: csiv1.IBMBlockCSIControllerSpec{
+					Repository: "fake-controller-repo",
+					Tag:        "fake-controller-tag",
+				},
+				Node: csiv1.IBMBlockCSINodeSpec{
+					Repository: "fake-node-repo",
+					Tag:        "fake-node-tag",
+				},
+				Sidecars: []csiv1.CSISidecar{
+					{
+						Name:            "csi-node-driver-registrar",
+						Repository:      "fake-registrar-repo",
+						Tag:             "fake-registrar-tag",
+						ImagePullPolicy: "IfNotPresent",
+					},
+					//{
+					//	Name:            "csi-provisioner",
+					//	Repository:      "k8s.gcr.io/sig-storage/csi-provisioner",
+					//	Tag:             "v2.1.0",
+					//	ImagePullPolicy: "IfNotPresent",
+					//},
+					//{
+					//	Name:            "csi-attacher",
+					//	Repository:      "k8s.gcr.io/sig-storage/csi-attacher",
+					//	Tag:             "v3.1.0",
+					//	ImagePullPolicy: "IfNotPresent",
+					//},
+					//{
+					//	Name:            "csi-snapshotter",
+					//	Repository:      "k8s.gcr.io/sig-storage/csi-snapshotter",
+					//	Tag:             "v3.0.2",
+					//	ImagePullPolicy: "IfNotPresent",
+					//},
+					//{
+					//	Name:            "csi-resizer",
+					//	Repository:      "k8s.gcr.io/sig-storage/csi-resizer",
+					//	Tag:             "v1.1.0",
+					//	ImagePullPolicy: "IfNotPresent",
+					//},
```

Comment:
?

Reply:
removed this file

Reply:
so we will lose all the `ibmblockcsi_controller_test.go` tests?

Reply:
no, you still have it in 1.6.0 tag in GitHub, when we will use it, we will copy it from there

Reply:
1. by "use" you mean re-add them in one of the tickets which you opened?
2. do they currently run in develop?
3. do we want to remove them because they are failing in this PR?

Reply:
1. yes
2. no
3. I removed them since we are not using them and you had comments on them